Makefile

```diff
@@ -91,28 +92,36 @@ install-plain: install-apis ## Install the rukpak CRDs and the plain provisioner
 install: install-plain ## Install all rukpak core CRDs and provisioners
 
 deploy: install-apis ## Deploy the operator to the current cluster
+	kubectl apply -f internal/core/manifests
 	kubectl apply -f internal/provisioner/plain/manifests
 
-run: build-local-container kind-load deploy ## Build image and run operator in-cluster
+run: build-local-container kind-load cert-mgr deploy ## Build image and run operator in-cluster
+
+cert-mgr:
```

Comment:
Add a comment about this target so it shows up in the `make help` output locally?

Reply:
I'll add it.  Thanks!